gitprompt.sh

```diff
@@ -10,6 +10,7 @@ function git_prompt_dir() {
   # assume the gitstatus.sh is in the same directory as this script
   # code thanks to http://stackoverflow.com/questions/59895
   if [[ -z "${__GIT_PROMPT_DIR:+x}" ]]; then
+    unset CDPATH
```

Comment:
@benvaljean wouldn't this unset the CDPATH for the whole bash-session where the `gitprompt.sh` is sourced from?